### PR TITLE
Fixes as suggested by Otto

### PR DESCRIPTION
**Pull request by janajaeger**
- apply Otto's feedback

### CODE REVIEW

**Review by jfaltenbacher — APPROVED** [review]
LGTM

File: references/sudo-user-authorization.xml [hunk 1]
```diff
@@ -45,7 +47,10 @@ tux ALL = NOPASSWD: /usr/bin/frobnicate, PASSWD: /usr/bin/journalctl<co xml:id="
         <para>
           There are two exceptions: <literal>#include</literal>
           and <literal>#includedir</literal> are regular commands.
-          <!--FIXME, jjaeger:  is this still true with the current version? Current SUSE has @includedir -->
+          More current version do not use the <literal>#</literal> any more.
+          Instead include directives are now preceded by <literal>@</literal>.
+          <literal>#</literal> notation is still supported for backwards
+          compatibility reasons.
```
Comment (dariavladykina) [inline references/sudo-user-authorization.xml:53]:
```suggestion
          The more current version does not use the <literal>#</literal> anymore.
          Instead, include directives are now preceded by <literal>@</literal>.
          The <literal>#</literal> notation is still supported for backward
          compatibility reasons.
```

File: tasks/sudo-without-root-password.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    <step>
+      <para>
+        As a system administrator, add the user to the
+        <systemitem class="groupname">wheel</systemitem> group:
+      </para>
+<screen>&prompt.sudo; <command>usermod -aG wheel <replaceable>USERNAME</replaceable></command></screen>
+    </step>
+    <step>
+      <para>
+        Check status of the user's membership of the
```
Comment (dariavladykina) [inline tasks/sudo-without-root-password.xml:50]:
```suggestion
        Check the status of the user's membership in the
```

File: tasks/sudo-without-root-password.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        <systemitem
+      class="groupname">wheel</systemitem> group by running
+        the following command:
+      </para>
+<screen>&prompt.user; <command>cut -d: -f1,4 /etc/group | grep wheel</command></screen>
+<screen>wheel:<replaceable>USERNAME</replaceable></screen>
+      <para>
+        Alternatively, you can check the output of the <command>id</command>
+        command from a new terminal session. For example, the username
+        <literal>tux</literal> has the following ids associated with it:
```
Comment (dariavladykina) [inline tasks/sudo-without-root-password.xml:60]:
```suggestion
        <literal>tux</literal> has the following IDs associated with it:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Pull request merged**